Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1135,7 +1142,9 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, minimu
     return tests, packs_to_install
 
 
-def get_from_version_and_to_version_bounderies(all_modified_files_paths: set, id_set: dict) -> Tuple[str, str]:
+def get_from_version_and_to_version_bounderies(all_modified_files_paths: set,
+                                               id_set: dict,
+                                               modified_metadata_list: set = None) -> Tuple[str, str]:
```

Comment:
```suggestion
                                               modified_metadata_list: set = set([])) -> Tuple[str, str]:
```

Reply:
no, that will be a bug

![image](https://user-images.githubusercontent.com/30797606/101752823-9ca0d680-3ada-11eb-9d11-9236a8ad23f4.png)

Reply:
:yaakovi: